content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_design_dialog/.content.xml

```diff
@@ -35,36 +35,64 @@
                         jcr:title="Properties"
                         sling:resourceType="granite/ui/components/coral/foundation/container"
                         margin="{Boolean}true">
-                        <items jcr:primaryType="nt:unstructured">
-                            <heading
+                        <items
+                            jcr:primaryType="nt:unstructured"
+                            sling:hideChildren="[heading,well]">
```

Comment:
@vladbailescu - note, I had to hide the heading and well nodes that contained the multifield and that we inherit from the basic page component.